router-tests/security/ratelimit_redis_pool_test.go

```
@@ -0,0 +1,131 @@
+
+	const (
+		poolSize   = 4
+		concurrent = 48
+		perWorker  = 15
+	)
+
+	key := uuid.New().String()
+
+	t.Cleanup(func() {
+		client := redis.NewClient(&redis.Options{Addr: "localhost:6379", Password: "test"})
+		defer client.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `client.Close` is not checked

(errcheck)